ginkgo.Jenkinsfile

```diff
@@ -85,7 +85,7 @@ pipeline {
                     // rm -rf ${GOPATH}/src/github.com/cilium/cilium
                     sh 'cd test/; ./post_build_agent.sh || true'
                     sh 'cd test/; ./archive_test_results.sh || true'
-                    archiveArtifacts artifacts: "test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.zip", allowEmptyArchive: true
```

Comment:
The change looks fine, but the commit message needs to be more descriptive than saying this addresses an 'issue'. Most commits address issues, so this is somewhat of a vacuous statement. Please be sure to state what issue is actually being addressed within the commit itself, as it makes the commit history easier to follow.